### DISCUSSION TITLE
Big green download button

### DISCUSSION THREAD

**Discussion by initiumx** [focustense/easymod#187] [Ideas]
I'd like to request a 'big green download' button, or something similar on Github.

The reason is that I fear Nexus is starting to transition towards a proprietary service provider, with an eventual goal of cashing in and floating on the market. Thus it would be good to have an alternative to downloading this mod beyond the confines of Nexus. At the very least it would be a warning to them that they need to remain a library of mods, rather than cashing in off the backs of other peoples talent.

**Comment by focustense** [comment]
What's the request here, exactly? Like most projects on GitHub, the release history is always available and the current release is displayed on the project home page with a "latest" tag.

This has always been the primary repository - Nexus gets updated whenever I've got the mental fortitude to try to get the Nexus Uploader working again.

**Reply by initiumx** [reply]
GitHub is a fantastic resource but isn't very clear or friendly for casual users. A 'Big Green Button' for downloading your great mod would be really useful for casual users that are not familiar with how GitHub works.  So my suggestion/idea, is merely to help people navigate and download in one click, your latest version of this mod. I hope this makes sense to you.

To support what I am suggesting consider that on the https://github.com/focustense/easymod/wiki/easynpc-%E2%80%90-Home page, there is nothing at the top which explains how to download, nor any GUI element. The top menu bar has 'Wiki' underlined next to 'Projects', and there is no "download" option in sight, leaving any unfamiliar with GitHub what to click or where to go. The info regarding where to download is halfway down the page. To really demonstrate how this GUI element is missing and confusing, consider that on this page I have below this box a small green button called 'Reply', which seems pretty intuitive. 

This isn't a fault of yours, rather that GitHub isn't designed for casual users looking to download content like a game mod, nor does it use the same generic phrasing for naming conventions of sites like Nexus. But if it is at all possible to place a GUI element such as a 'Big Green Button' for downloading at the top of the page, it would be really useful. Perhaps you could set this up as a resource for other modders on GitHub, so they too can use it.

I hope that helps clarify what I mean. If it's not something that you wish to pursue that's fine, perhaps someone else will come up with it as a resource in the future which can be adopted across GitHub. In which case I'd like to suggest that you add a contents section that navigates to 'Installation' so people know they're in the right place on GitHub.
All the best focustense, and thank you for EasyNPC.
